packages/shared/src/contexts/AuthContext.tsx

```diff
@@ -1,8 +1,8 @@
 import React, {
   ReactElement,
   ReactNode,
+  useCallback,
```

Comment:
Changes here are unnecessary, but we at least removed one `eslint-disable-next-line react-hooks/exhaustive-deps`